app/models/concerns/stat/amount_consumed.rb

```diff
@@ -0,0 +1,74 @@
+class Stat
+  module AmountConsumed
+    extend ActiveSupport::Concern
+
+    DEFAULT_STATS = {
+      'total_media' => 0,
+      'total_progress' => 0,
+      'total_time' => 0
+    }.freeze
+
+    # Fully regenerate data
+    def recalculate!
+      entries = user.library_entries.eager_load(media_column)
+                    .where.not(media_length => nil)
+
+      stats_data['all_time'] = {
+        total_media: entries.count, # all anime or manga
+        total_progress: entries.sum(:progress), # all episodes or chapters
+        total_time: entries.sum(:time_spent) # time spent (anime only)
+      }
+
+      save!
+    end
+
+    class_methods do
```

Comment:
I'm kinda tempted to disable this cop, simply because it's pretty much ALWAYS a false positive